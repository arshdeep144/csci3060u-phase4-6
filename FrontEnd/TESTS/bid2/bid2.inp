login
user14
bid
logout
