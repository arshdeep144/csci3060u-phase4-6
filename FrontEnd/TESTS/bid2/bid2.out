Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
You do not have access to this feature
